spec/core/block/routes/getMerkleTransaction.yml

```diff
@@ -3,9 +3,16 @@ tags:
 summary: Get the merkle path for a given a transaction and block
 description: |
   Returns the merkle path for a transaction included in a block.
-  The path is the complementary data needed to calculate the merkle root.
-  A client can compare if the calculated root equals the one recorded in the
-  block header, verifying that the transaction was included in the block.
+  The merkle path is the minimum number of nodes needed to calculate the merkle root.
+
+  Steps to calculate the merkle root:
+  1. proofHash = hash (leaf)
+  2. Contatentate and hash the first item from the merklePath list:
+  * a) If item.position == left -> proofHash = sha_256(item.hash + proofHash)
+  * b) If item.position == right -> proofHash = sha_256(proofHash+ item.hash)
+  3. Repeat 2. for every item in the merkle_path list
```

Comment:
merklePath or merkle_path, but consistent with previous/future apperances (seems to be camelCasing)